### PR TITLE
Add swap value example to documentation

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #124, #125 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Create an example that demonstrates how to make two `auro-combobox` swap values using the new icon only `auro-button`
* Update peer dependencies to direct dependencies of the child Auro components to make the examples on the docsite work

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/auro-combobox.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ import { LitElement, html } from "lit-element";
 /* eslint-disable max-lines */
 
 import '@aurodesignsystem/auro-menu';
+import '@aurodesignsystem/auro-input';
+import '@aurodesignsystem/auro-dropdown';
```
[the remark this answers is not in the document]
Comment (blackfalcon) [reply src/auro-combobox.js:18]: Please update footer of this document to match this 

https://github.com/AlaskaAirlines/auro-badge/blob/main/src/auro-badge.js#L114

File: index.js [hunk 2]
```diff
@@ -1 +1,13 @@
...
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+export function registerComponent(name) {
+  // alias definition
+  if (!customElements.get(name)) {
+    customElements.define(name, class extends AuroCombobox {});
+  }
+}
```
Comment (blackfalcon) [inline index.js:13]: You need to update rollup config, see file

- https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/rollup.config.mjs

Also be sure to update the .npmignore to include the proper resource from the ./src dir.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: demo/index.html [hunk 3]
```diff
@@ -32,27 +32,20 @@
...
 
     <!-- If additional elements are needed for the demo, add them here. -->
     <script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.js" type="module"></script>
     <script src="https://unpkg.com/@alaskaairux/auro-icon@latest/dist/auro-icon__bundled.js" type="module"></script>
-    <script src="https://unpkg.com/@aurodesignsystem/auro-dropdown@latest/dist/auro-dropdown__bundled.js" type="module"></script>
-    <script src="https://unpkg.com/@aurodesignsystem/auro-input@latest/dist/auro-input__bundled.js" type="module"></script>
     <script src="https://unpkg.com/@alaskaairux/auro-button@latest/dist/auro-button__bundled.js" type="module"></script>
-    <script type="module" src="../src/auro-combobox.js"></script>
+    <script type="module" src="../index.js"></script>
+    <script type="module" src="./index.js"></script>
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply demo/index.html:49]: * `../index.js` refers to the file that handles defining the component
* `./index.js` refers to running the javascript examples on the `demo.md` page

File: test/auro-combobox.test.js [hunk 4]
```diff
@@ -506,7 +507,7 @@ async function noFilterFixture() {
 }
 
 function setInputValue(el, value) {
-  const auroInput = el.shadowRoot.querySelector('auro-input');
+  const auroInput = el.shadowRoot.querySelector('combobox-input');
```
Comment (jason-capsule42) [inline test/auro-combobox.test.js:510]: we need to preserve `auro` in the component names.

Let's rename all these to `auro-comboboxinput`

File: test/auro-combobox.test.js [hunk 5]
```diff
@@ -153,7 +150,7 @@ describe('auro-combobox', () => {
   it('navigates menu with up and down arrow keys', async () => {
     const el = await defaultFixture();
 
-    const dropdown = el.shadowRoot.querySelector('auro-dropdown');
+    const dropdown = el.shadowRoot.querySelector('combobox-dropdown');
```
Comment (jason-capsule42) [inline test/auro-combobox.test.js:153]: we need to preserve `auro` in the component names.

Let's rename all these to `auro-comboboxdropdown`

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: test/auro-combobox.test.js (hunk 4 above)
Comment (jordanjones243) [reply test/auro-combobox.test.js:510]: > we need to preserve `auro` in the component names.
> 
> Let's rename all these to `auro-comboboxinput`

I made these changes as per @blackfalcon's request. We can discuss as a team which naming convention we would like.

File: test/auro-combobox.test.js (hunk 5 above)
Comment (jordanjones243) [reply test/auro-combobox.test.js:153]: Please see comment above.

**Review by jason-capsule42 — DISMISSED** [review]

File: package.json [hunk 6]
```diff
@@ -26,14 +26,12 @@
...
-    "@aurodesignsystem/auro-dropdown": "^2.9.1",
-    "@aurodesignsystem/auro-input": "^2.10.2",
     "@aurodesignsystem/webcorestylesheets": "^4.0.0"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.12.1",
-    "@aurodesignsystem/auro-dropdown": "^2.9.4",
-    "@aurodesignsystem/auro-input": "^2.10.4",
+    "@aurodesignsystem/auro-dropdown": "^2.9.7",
+    "@aurodesignsystem/auro-input": "^2.13.2",
```
Comment (blackfalcon) [inline package.json:34]: These need to be direct dependencies. As dev dependencies, they will not be included in the bundle and not be available to be used for the extension.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**